src/Stratis.Bitcoin.Features.BlockStore/StoreSettings.cs

```diff
@@ -56,10 +60,11 @@ public static void PrintHelp(Network network)
         {
             var builder = new StringBuilder();
 
-            builder.AppendLine($"-txindex=<0 or 1>         Enable to maintain a full transaction index.");
-            builder.AppendLine($"-reindex=<0 or 1>         Rebuild chain state and block index from block data files on disk.");
-            builder.AppendLine($"-prune=<0 or 1>           Enable pruning to reduce storage requirements by enabling deleting of old blocks.");
-            builder.AppendLine($"-maxCacheBlocksCount=<number> The maximum amount of blocks the cache can contain. Default is {DefaultMaxCacheBlocksCount}.");
+            builder.AppendLine($"-txindex=<0 or 1>              Enable to maintain a full transaction index.");
+            builder.AppendLine($"-reindex=<0 or 1>              Rebuild chain state and block index from block data files on disk.");
+            builder.AppendLine($"-prune=<0 or 1>                Enable pruning to reduce storage requirements by enabling deleting of old blocks.");
+            builder.AppendLine($"-pruneblockmargin=<2880 +>     The amount of blocks to keep on disk.");
```

Comment:
I suggest instead of pruneblockmargin  we just use the prune value and instead of bool it will be the number of blocks to prune or 0/-1 disabled

Reply:
OK yeah let me fix this 👍

Reply:
Bitcoin does it that way, just for consistency check how do they disable pruning

Reply:
Bitcoin regards -prune=0 as disabled. I've made the changes.